documentation/docs/strategies/cross-exchange-market-making.md

```diff
@@ -48,9 +58,12 @@ The following walks through all the steps when running `config` for the first ti
 | `Enter the token symbol you would like to trade on [maker exchange name] >>>`: | Enter the token symbol for the *maker exchange*.<br/>Example input: `ZRX-WETH`<br/><table><tbody><tr><td bgcolor="#ecf3ff">**Note**: ensure that the pair is a valid pair for the exchange, for example, use `WETH` instead of `ETH`.</td></tr></tbody></table> |
 | `Enter the token symbol you would like to trade on [taker exchange name] >>>`: | Enter the token symbol for the *taker exchange*.<br/>Example input: `ZRX-ETH`<br/><table><tbody><tr><td bgcolor="#ecf3ff">**Note**: ensure that (1) the pair corresponds with the token symbol entered for the maker exchange, and that (2) is a valid pair for the exchange.  *Note in the example, the use of `ETH` instead of `WETH`*.</td></tr></tbody></table>|
 | `What is the minimum profitability for your to make a trade? (Enter 0.01 to indicate 1%) >>>`: | This sets `min_profitability` (see [definition](/strategies/cross-exchange-market-making/#configuration-parameters)). |
+| `Do you want to adjust the prices to be above top bid/ask instead of the expected price, if profitable ? (Default is True) >>>`: | This sets `adjust_order_enabled` (see [definition](/strategies/cross-exchange-market-making/#configuration-parameters)). |
```

Comment:
confusing, what is expected price? what does it mean by "if profitable"? will hummingbot place orders if it's not profitable?

Reply:
If its profitable to place above top bid/ask it will place it there. How do you want to change this ?

Reply:
no blank space before `?`